pulsar-client/src/main/java/org/apache/pulsar/client/impl/AuthenticationUtil.java

```diff
@@ -43,9 +43,10 @@ public static Map<String, String> configureFromPulsar1AuthParamString(String aut
         if (isNotBlank(authParamsString)) {
             String[] params = authParamsString.split(",");
             for (String p : params) {
-                String[] kv = p.split(":");
-                if (kv.length == 2) {
-                    authParams.put(kv[0], kv[1]);
+                // The value could be a file path, which could contain a colon like "C:\\path\\to\\file" on Windows.
```

Comment:
Could you add a unit test?

Reply:
Added now. I also modified a little to filter empty keys or values. PTAL again.

Reply:
@nodece PTAL again

Reply:
I suggest we should support empty values.

Reply:
@nodece I agree since the previous implementation also allows empty values. It's addressed now.